projects/continuous_learning/sync_ec2_repo.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Infrastructure stuff shouldn't be in this rep. In addition, the stuff here is specific to afisher. Please remove this file from here.